controller/auth/user.go

```diff
@@ -36,7 +36,7 @@ type AuthUser struct {
 	Password string
 }
 
-type AuthResponse struct {
+type authResponse struct {
 	Token string
```

Comment:
Types are great! You can make this more ~enterprise by doing:
```
type JWToken string; // this can also be []byte
type authResponse struct {
        Token JWToken
}
```
You'd then you'll just make `JWToken`s with `JWToken(serializedJWT)`

This will make your types more semantic and harder to mix up (and the compiler can check!).